atest/testdata/running/for.robot

```diff
@@ -40,6 +40,13 @@ For Loop Over Empty List Variable Is Ok
     \    Fail    not executed
     Variable Should Not Exist    ${var}
 
+For Loop Over Generator
+    ${xrange}=    Evaluate    xrange(10)
```

Comment:
Considering the future Python 3 support, it would be better to use, for example, `(i for i in 'abcdefg')` instead of `xrange`.